colossalai/engine/ophooks/_memtracer_ophook.py

```diff
@@ -121,11 +123,22 @@ def __init__(self, warmup: int = 50, refreshrate: int = 10, data_prefix: str = "
             self._rank = gpc.get_global_rank()
         else:
             self._rank = 0
+        self._logger.debug("initialized MemTracerOpHook")
```

Comment:
Can remove this?

Reply:
I think this may be useful when we want to know whether this hook has been activated. Any specific reasons on removing this?